app/controllers/cards_controller.rb

```diff
@@ -1,5 +1,48 @@
 class CardsController < ApplicationController
+
+  before_action :find_card, only: [:update, :edit, :destroy]
+
   def index
     @card = Card.all
   end
+
+  def new
+    @card = Card.new
+  end
+
+  def create
+    @card = Card.new(card_params)
+
+    if @card.save
+      redirect_to cards_path
+    else
+      render 'new'
+    end
+  end
+
+  def update
+    if @card.update(card_params)
+      redirect_to cards_path
+    else
+      render 'edit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.